addons/web/static/src/legacy/utils.js

```diff
@@ -78,22 +78,22 @@ export function makeLegacyDialogMappingService(legacyEnv) {
                 return dialog.modalRef ? () => dialog._close() : () => dialog.$modal.modal("hide");
             }
 
-            const tokensMap = new Map();
+            const hotkeyRemoveMap = new Map();
```

Comment:
why is this a `Map<removeFn, Dialog>` instead of `Map<Dialog, removeFn[]>`? this seems totally backward